IdentityCore/src/MSIDConstants.m

```diff
@@ -31,6 +31,7 @@
 NSString *const MSID_DEVICE_MODEL_KEY              = @"x-client-DM";
 NSString *const MSID_APP_NAME_KEY                  = @"x-app-name";
 NSString *const MSID_APP_VER_KEY                   = @"x-app-ver";
+NSString *const MSID_CCS_HINT_KEY                  = @"X-AnchorMailbox";
```

Comment:
Interesting name for an auth parameter.... is this right?

Reply:
yep, it is correct :)